tests/Neo.Plugins.OracleService.Tests/UT_OracleService.cs

```diff
@@ -16,6 +16,46 @@ public void TestSetup()
             TestBlockchain.InitializeMockNeoSystem();
         }
 
+        [TestMethod]
+        public void TestFilter()
+        {
+            var json = @"{
+  'Stores': [
+    'Lambton Quay',
+    'Willis Street'
+  ],
+  'Manufacturers': [
+    {
+      'Name': 'Acme Co',
+      'Products': [
+        {
+          'Name': 'Anvil',
+          'Price': 50
+        }
+      ]
+    },
+    {
+      'Name': 'Contoso',
+      'Products': [
+        {
+          'Name': 'Elbow Grease',
+          'Price': 99.95
+        },
+        {
+          'Name': 'Headlight Fluid',
+          'Price': 4
+        }
+      ]
+    }
+  ]
+}";
+
+            Assert.AreEqual(@"[""Acme Co""]", Utility.StrictUTF8.GetString(OracleService.Filter(json, "Manufacturers[0].Name")));
+            Assert.AreEqual("[50]", Utility.StrictUTF8.GetString(OracleService.Filter(json, "Manufacturers[0].Products[0].Price")));
+            Assert.AreEqual(@"[""Elbow Grease""]", Utility.StrictUTF8.GetString(OracleService.Filter(json, "Manufacturers[1].Products[0].Name")));
+            Assert.AreEqual(@"[{""Name"":""Elbow Grease"",""Price"":99.95}]", Utility.StrictUTF8.GetString(OracleService.Filter(json, "Manufacturers[1].Products[0]")));
```

Comment:
should add Filter(json,"") and Filter(json,null)

Reply:
I'll add it.

Reply:
https://github.com/neo-project/neo-modules/blob/0acf93b223e8510530056625823e1a9363cf6961/src/OracleService/OracleService.cs#L427-L428

It returns the unmodified input.